src/index.js

```diff
@@ -333,6 +334,16 @@ cronofy.prototype.refreshAccessToken = function () {
   }), details.callback);
 };
 
+cronofy.prototype.hmacMatch = function () {
```

Comment:
Might rename this - `hmacValid` or `hmacMatches` feel a bit more descriptive? Not sure, what do you think?
```suggestion
cronofy.prototype.hmacValid = function () {
```

Reply:
Yeah, no probs. Let me change it!

Reply:
Change applied! `hmacMatch` renamed as `hmacMatches`